### PR TITLE
Update docs with LF processes

### PR DESCRIPTION
**Pull request by marmbrus**
This PR updates our docs to discuss the new process for accepting commits under the Linux foundation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ To generate artifacts, run
     build/sbt package
 
 To execute tests, run
-
```
Comment (marmbrus) [inline README.md]: There was a bunch of garbage whitespace here that my editor removed...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,74 @@
 We happily welcome contributions to Delta Lake. We use [GitHub Issues](/../../issues/) to track community reported issues and [GitHub Pull Requests ](/../../pulls/) for accepting changes.
+
+# Governance
+Delta lake governance is conducted by the Technical Steering Committee (TSC), which is currently composed of the following members:
+ - Michael Armbrust (michael.armbrust@gmail.com)
+ - Reynold Xin (reynoldx@gmail.com)
+ - Matei Zaharia (matei@cs.stanford.edu)
+
+The founding technical charter can be found [here]().
```
Comment (tdas) [inline CONTRIBUTING.md]: link?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Delta lake governance is conducted by the Technical Steering Committee (TSC), wh
  - Reynold Xin (reynoldx@gmail.com)
  - Matei Zaharia (matei@cs.stanford.edu)
 
-The founding technical charter can be found [here]().
+The founding technical charter can be found [here](https://delta.io/wp-content/uploads/2019/12/delta-charter.pdf).
```
Comment (marmbrus) [inline CONTRIBUTING.md]: Thank you!

**Comment by ekoifman** [comment]
Hi,
is there an ETA when this may be merged?

**Pull request closed without merging**